core/services/ocr2/delegate.go

```diff
@@ -1013,7 +1015,7 @@ func (d *Delegate) newServicesDKG(
 			JobID:         jb.ID,
 			ContractID:    spec.ContractID,
 			New:           d.isNewlyCreatedJob,
-			RelayConfig:   spec.RelayConfig.Bytes(),
+			RelayConfig:   d.relayConfigsWithDefaults(spec.RelayConfig),
```

Comment:
Would it be possible to do this just once in the caller `ServicesForSpec`, before the `PluginType` switch?
Seems like it should work, and then `relayConfigsWithDefaults()` can just update the `job.JSONConfig` instead of returning `.Bytes()`?